### DISCUSSION TITLE
When i use ItemsWrapGrid for ListView，the

### DISCUSSION THREAD

**Discussion by LionelXN** [unoplatform/uno#11928] [Q&A]
I added five items, but only displayed one item of data. Why is that?
![image](https://user-images.githubusercontent.com/18021306/230547109-530617d2-7e0a-40e8-9166-e6f66d81c2f8.png)
![image](https://user-images.githubusercontent.com/18021306/230547780-303f2387-04ee-4a52-8794-cfc9b1cea38d.png)

**Comment by LionelXN** [comment]
I hope to use a virtualized list and support the presentation of data in multiple columns. Perhaps, is there any other way to meet this requirement?

**Reply by jeromelaban** [reply]
See https://github.com/unoplatform/uno/issues/4023

**Reply by LionelXN** [reply]
Thanks for your reply. I tried and it still doesn't work with Gtk

**Reply by jeromelaban** [reply]
Can you provide what you tried? Can you try making a sample [in the playground](https://playground.platform.uno/)?
